Measurement/twinkles_light_curves.ipynb

```diff
@@ -0,0 +1,480 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import lsst.afw.display as afw_display\n",
+    "from lsst.daf.persistence import Butler\n",
+    "\n",
+    "from lsst.geom import SpherePoint, Angle, degrees\n",
+    "import lsst.geom as geom\n",
+    "import lsst.afw.image as afwImage\n",
+    "\n",
+    "from astropy.coordinates import SkyCoord\n",
+    "from astropy import units as u\n",
+    "\n",
+    "from matplotlib import pyplot as plt\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Find Twinkles Lensed AGN\n",
+    "\n",
+    "We have saved the Twinkles truth locations for lensing galaxies and the AGN images. Here we load them in with a method so that it can be used in other stack club notebooks."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get twinkles object locations\n",
+    "# Make it a method so we can use it in other notebooks\n",
+    "def get_twinkles_truth():\n",
+    "    lens_truth = pd.read_csv('/project/shared/data/Twinkles_subset/truth/sprinkled_lens_230_J2000.txt')\n",
+    "    agn_truth = pd.read_csv('/project/shared/data/Twinkles_subset/truth/sprinkled_agn_230_J2000.txt')\n",
+    "    return lens_truth, agn_truth"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "twinkles_lens_truth, twinkles_agn_truth = get_twinkles_truth()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Visualize one of these systems in the Twinkles data\n",
+    "\n",
+    "Sanity check our truth catalog by visualizing one of the systems. To do this we will use the `butler` to load a `calexp` image."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set up a butler\n",
+    "datadir = '/project/shared/data/Twinkles_subset/output_data_v2/'\n",
+    "butler = Butler(datadir)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get the data for one visit\n",
+    "calexp_subset = butler.subset('calexp')\n",
+    "dataId = calexp_subset.cache[0]\n",
+    "print(dataId)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get the calexp\n",
+    "calexp = butler.get('calexp', **dataId)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can use `afw_display` to load the calexp image and display it using a `matplotlib` backend."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig = plt.figure(figsize=(8,8))\n",
```

Comment:
I didn't know this worked.  Is it because the display routines use a variable called `fig` so this is somehow making it into the scope of the display class, or is this magic that is specific to matplotlib?

Reply:
Turns out this isn't actually doing anything.

Reply:
Oh wait, this actually does change the image size. Hmm...I'm thinking it must be related to matplotlib and the ipython magic `%matplotlib inline`.

Reply:
FYI The version of `display_matplotlib` that I need to push supports passing a `figure` to `Display` (so you can use subplots)